tpgtools/main.go

```diff
@@ -30,11 +31,16 @@ import (
 
 	"github.com/nasa9084/go-openapi"
 	"gopkg.in/yaml.v2"
+
+	// Used by serialization.go.tmpl, included here to keep `go mod tidy` from removing
+	// the dependency from go.mod.
+	_ "github.com/hashicorp/hcl/hcl/fmtcmd"
 )
 
 var fPath = flag.String("path", "", "to be removed - path to the root service directory holding samples")
 var tPath = flag.String("overrides", "", "path to the root directory holding overrides files")
 var cPath = flag.String("handwritten", "handwritten", "path to the root directory holding handwritten files to copy")
+var gPath = flag.String("templates", "templates", "path to the templates directory")
```

Comment:
would it be better to just embedd the template file instead of try to find it?

Reply:
I don't know if it's better or worse, I just I did it this way to match the --path and --overrides flags that are already needed in the genrule. I can switch if you feel strongly about it (I don't have a strong opinion either way).